public/stylesheets/site/2.0/13-group-blurb.css

```diff
@@ -357,6 +357,10 @@ blurbs on the manage collection items pages, mostly reseting styles inherited fr
   clear: both;
 }
 
+.bookmark .blurb dl.stats {
```

Comment:
Does just  `.bookmark dl.stats` happen to work, just to bring it more in line with the other bookmark selectors?